### DISCUSSION TITLE
Custom image size of Image component via className does not works

### DISCUSSION THREAD

**Discussion by tony9527167** [mantinedev/mantine#3629] [Q&A]
https://codesandbox.io/s/pensive-bogdan-k4unls?file=/src/App.tsx

**Comment by yverby** [comment]
According to the [documentation](https://mantine.dev/core/image/), `Image` component relies on its parent, otherwise you need to specify `width` and `height` props, or you can add `!important` to the width or height in your styles values as a workaround
